gulpfile.js

```diff
@@ -105,7 +105,7 @@ gulp.task('transpile', ['clean-internal'], () => {
       presets: ['es2015'],
     }))
     .on('error', (err) => { console.error(err); }) // eslint-disable-line
-    .pipe(replace(/process\.env\.(.+)\;/gi, (envCall, envKey) => {
+    .pipe(replace(/process\.env\.([^\.]+)\;/gi, (envCall, envKey) => {
```

Comment:
My regexing skillz aren't the best but this doesn't change anything right?  It simply blocks

`process.env..SOMETHING`?